test/testdata/packager/lsp_features/__package.rb

```diff
@@ -0,0 +1,15 @@
+# typed: strict
+# enable-packager: true
+
+class Simpsons < PackageSpec
+    # ^^^^^^^^ symbol-search: "Simpsons", name = "Simpsons", container = "(nothing)"
+    # go-to-def on a reference to Bart within the package goes here, but go-to-def on Bart in `import Bart` goes to the
+    # package.
+    import Bart
+    #      ^^^^ symbol-search: "Bart", name = "Bart", container = "Simpsons"
+    #      ^^^^ def: bart 1 not-def-of-self
+    #      ^^^^ usage: bartpkg
+    #      ^^^^ hover: Bart package description
```

Comment:
So there was basically no additional work to get LSP working for packaged files? Just some small changes to how position assertions work?

Reply:
Yup! Since packager is such a dead-simple syntactic transformation, and since Sorbet ~erases the aliases it sets up, everything more-or-less "just works".

The following had to line up for LSP features to work properly:

* The synthetic namespaces/modules/aliases created in packager needed proper `loc` information. I already took care of this in the previous PR, as this property is required for good error messages!
* Synthetic stuff that shouldn't appear in LSP query results should not have a `loc` set. Thankfully, we were careful with locs in the previous PR.
* The slow path needs to trigger when a package is updated. That's handled with a simple "has the source text changed" check.
* The fast path needs all packaged files passed to it for `incrementalPackager` to work properly. This was probably the biggest change, as it means querying `foo.rb` can return results in `__package.rb` files!

For a nicer UX:
* Pretty printed types had to hide the synthetic `<PackageRegistry>` namespace, and display the package name when appropriate.